src/components/InteractionCanvas.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div id="interaction-canvas"></div>
+</template>
+
+<script setup>
+import { onMounted } from "vue";
+
+/* eslint-disable */
+onMounted(() => {
+  // NOTE: Use p5 as an instance mode
+  const P5 = require("p5");
+  const script = function (p5) {
+    const P5_CANVAS = {
+      width: 800,
+      height: 800
```

Comment:
set dynamic size according the container? 
something like:

```suggestion
      width: windowWidth,
      height: windowHeight
```

(not tested)

Reply:
smart